backend/typescript/rest/reviewRoutes.ts

```diff
@@ -59,9 +59,12 @@ reviewRouter.get(
   "/",
   isAuthorizedByRole(new Set(["Admin", "Subscriber", "Author"])),
   async (req, res) => {
+    const searchTerm = req.query.search;
     const contentType = req.headers["content-type"];
     try {
-      const reviews = await reviewService.getReviews();
+      const reviews = searchTerm
+        ? await reviewService.getReviewsWithSearch(searchTerm as string)
```

Comment:
Think it might be better to have the function getReviews func take optional params? We actually also hav page and size query params that hav now been merged into development so might hav to change this a bit when rebasing

Reply:
fixed